framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -92,38 +93,54 @@ public static void main(String[] args) throws Exception {
     }
 
     final Stopwatch stopwatch = Stopwatch.createStarted();
-    int errorCode;
+    int errorCode=0;
+    Date startDate =null;
     try {
-      LOG.info(LINE_BREAK + LINE_BREAK);
-      LOG.info("STARTING AT " + new Date());
-      LOG.info(LINE_BREAK);
+      startDate = new Date();
       errorCode = new TestDriver().runTests();
     } catch (Exception e) {
       LOG.error("Exiting due to uncaught exception", e);
       errorCode = -1;
     } finally {
+      LOG.info("\n" + LINE_BREAK);
+      LOG.info("RUN STATUS");
+      LOG.info(LINE_BREAK);
+      LOG.info("Exit Code      : " + errorCode);
+      LOG.info("Exit Status    : " + ((errorCode==0) ? "SUCCESS" : "FAILURE"));
+      LOG.info("\nRun Started    : " + startDate);
+      LOG.info("Run Completed  : " + new Date());
+      LOG.info("Total Duration : " + stopwatch);
```

Comment:
Updated the test run status to include error code, status. And updated the positioning and formatting.